modules/crud.js

```diff
@@ -0,0 +1,87 @@
+import Todo from './todo.js';
+
+const mainList = document.querySelector('.tasks');
+
+class TodoList {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('list')) || [];
+    this.displayList();
+    this.enterEvent();
+    this.indexes();
+  }
+
+  add(text) {
+    const newTodo = new Todo(text);
+    newTodo.index = this.list.length;
```

Comment:
* As the previous CR stated your index if you're adding a 5th task to the list, the index of that task should equal 5 for example first todo == index 1, fourth todo == index 4, kindly fix this same thing also should be done when deleting if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1